### ISSUE TITLE
Unresolved dependency found during sorting - activesupport (>= 4.2.0) (requested by rails-dom-testing-2.0.3)

### ISSUE THREAD

**Issue by diegobiavati** [rails/web-console#278]
After the Rails update, the gem web-console displays the following error when I try to upgrade:
```
$ gem update web-console
ERROR:  While executing gem ... (Gem::DependencyError)
    Unresolved dependency found during sorting - activesupport (>= 4.2.0) (requested by rails-dom-testing-2.0.3)
```

**Comment by mellansquare** [comment]
I'm also having the same issue. Did you find a solution diegobiavati?

**Comment by faqndo97** [comment]
I'm having the same issue with other gem, if you know something about how to fix it let us know!!!

**Comment by gsamokovarov** [comment]
Are you sure `web-console` is the culprit of the issue here and not the state of the Gemfile? Can you folks give me a bit more context about the environment in which the error occurs?

**Comment by mellansquare** [comment]
Hi,
I've been migrating my project from c9.io to AWS Cloud9 (because I can't install locally on my work computer) and in updating my gems I get this error. 

When I run gem update I get the following output in the terminal
```
Updating installed gems
Updating capybara
Fetching: capybara-3.20.1.gem (100%)
ERROR:  Error installing capybara:
        There are no versions of capybara (= 3.20.1) compatible with your Ruby & RubyGems
        capybara requires Ruby version >= 2.4.0. The current ruby version is 2.3.0.
Updating did_you_mean
ERROR:  Error installing did_you_mean:
        There are no versions of did_you_mean (= 1.3.0) compatible with your Ruby & RubyGems
        did_you_mean requires Ruby version >= 2.5.0. The current ruby version is 2.3.0.
Updating ffi
Fetching: ffi-1.11.0.gem (100%)
Building native extensions. This could take a while...
Successfully installed ffi-1.11.0
Updating fog
Building native extensions. This could take a while...
ERROR:  Error installing fog:
        ERROR: Failed to build gem native extension.

    current directory: /home/ubuntu/.rvm/gems/ruby-2.3.4/gems/ovirt-engine-sdk-4.3.0/ext/ovirtsdk4c
/home/ubuntu/.rvm/rubies/ruby-2.3.4/bin/ruby -r ./siteconf20190520-3868-xqphxs.rb extconf.rb
checking for xml2-config... yes
checking for curl-config... no
*** extconf.rb failed ***
Could not create Makefile due to some reason, probably lack of necessary
libraries and/or headers.  Check the mkmf.log file for more details.  You may
need configuration options.

Provided configuration options:
        --with-opt-dir
        --without-opt-dir
        --with-opt-include
        --without-opt-include=${opt-dir}/include
        --with-opt-lib
        --without-opt-lib=${opt-dir}/lib
        --with-make-prog
        --without-make-prog
        --srcdir=.
        --curdir
        --ruby=/home/ubuntu/.rvm/rubies/ruby-2.3.4/bin/$(RUBY_BASE_NAME)
        --with-libcurl-config
        --without-libcurl-config
        --with-pkg-config
        --without-pkg-config
extconf.rb:40:in `<main>': The "libcurl" package isn't available. (RuntimeError)

To see why this extension failed to compile, please check the mkmf.log which can be found here:

  /home/ubuntu/.rvm/gems/ruby-2.3.4/extensions/x86_64-linux/2.3.0/ovirt-engine-sdk-4.3.0/mkmf.log

extconf failed, exit code 1

Gem files will remain installed in /home/ubuntu/.rvm/gems/ruby-2.3.4/gems/ovirt-engine-sdk-4.3.0 for inspection.
Results logged to /home/ubuntu/.rvm/gems/ruby-2.3.4/extensions/x86_64-linux/2.3.0/ovirt-engine-sdk-4.3.0/gem_make.out
Updating web-console
ERROR:  Error installing web-console:
        Unresolved dependency found during sorting - activesupport (>= 4.2.0) (requested by rails-dom-testing-2.0.3)
Gems updated: capybara did_you_mean ffi ovirt-engine-sdk
```

Please let me know exactly what further information you require, as a learner-coder I'm never quite sure what info will be relevant to help answer my questions.

**Comment by diegobiavati** [comment]
> 
> 
> Are you sure `web-console` is the culprit of the issue here and not the state of the Gemfile? Can you folks give me a bit more context about the environment in which the error occurs?

My problem is not connected to Gemfile, I have a list of gems installed and I only requested to update all, and the error was presented, when I uninstalled the gem and I give a gem update I have no return of any error.

**Comment by lrjbrual** [comment]
I have the same issue. is anyone has a solution? thanks

**Comment by NfoCipher** [comment]
web-console 4.0 seems to depend on activesupport (6.0.0.rc1). If you specify gem install web-console -v3.7.0 , things seem to work ok with rails 5.2.3. Using ruby 2.6.3 here.

**Comment by tan-wei** [comment]
Waiting for the solution ...

**Comment by diegobiavati** [comment]
Issue resolved in version 4.0.1
